main.cpp

```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
+    int secondNumber = 3;
+    int thirdNumber = 4;
+    char firstInitial = 66;
+    char lastInitial = 66;
+    char middleInitial = 66;
+    bool thisIsTrue = true;
+    bool thisIsFalse = false;
+    bool isCorrect = true;
+    float firstFloat = 10.0f;
+    float secondFloat = 1.f;
+    float thirdFloat = 10.f;
+    double firstDouble = 10.44449;
+    double secondDouble = 11.5;
+    double thirdDouble = 12.000;
     
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(secondNumber);
+    ignoreUnused(thirdNumber);
+    ignoreUnused(firstInitial);
+    ignoreUnused(lastInitial);
+    ignoreUnused(middleInitial);
+    ignoreUnused(thisIsTrue);
+    ignoreUnused(thisIsFalse);
+    ignoreUnused(isCorrect);
+    ignoreUnused(firstFloat);
+    ignoreUnused(secondFloat);
+    ignoreUnused(thirdFloat);
+    ignoreUnused(firstDouble);
+    ignoreUnused(secondDouble);
+    ignoreUnused(thirdDouble);
 }
 /*
  10 functions
  example:
  */
 bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
+    { 
     ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
-
-/*
- 1)
- */
-
-/*
- 2)
- */
-
-/*
- 3)
- */
+    } 
+
+//Function 1
+int yearsWorking(int job1 = 4, int job2 = 10)
+    { 
+    ignoreUnused(job1, job2);
+    return {}; //function returns int.
+    }
+
+//Function 2
+void paintCar(int color, double howMuchPaint)
+    { 
+    ignoreUnused(color, howMuchPaint); // Do I need to include this or just remove this because it returns void?
+    }
+
+//Function 3
+bool careForDog(int hasWater = 1, int hasFood = 0, int hasWalk = 0)
+    { 
+    ignoreUnused(hasWater, hasFood, hasWalk);
+    return {}; //function returns bool.
+    }
```

Comment:
Fix your `{ }` placement.  see the style guide in Readme.MD if you are unsure where they should go.
- [ ] revise

Reply:
Thank you for being thorough! Will do.